### PR TITLE
Fix submit exceptions button

### PR DESCRIPTION
**Pull request by tausifpatel**
### Disable or enable the button in following scenarios to prevent users from submitting cases with empty editable fields when exceptions occur.

[**User story** 
](https://dsdmoj.atlassian.net/jira/software/c/projects/BICAWS7/boards/1368?selectedIssue=BICAWS7-2505)

Make submit exception(s) button disabled:
- When all the editable fields are empty
- When one of the editable fields is empty

Make submit exception(s) button enabled:
- When all the editable fields are filled in.
- For ASN, the entered value should be valid.
- If an ASN exception is not raised but any other exception occurs, the ASN editable field can be left empty when submitting the case.

### CODE REVIEW

File: src/utils/editableFieldsValidationError.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+// return true when field is empty
+// return true when invalid value is entered
+
+///// If there is no exception
+// return false when field is empty
+// return true when invalid value is entered
+
+// if (amendments.asn) {
+//   return !isAsnFormatValid(amendments?.asn as string) || isEmptyOrWhiteSpace(amendments?.asn as string)
+// }
```
Comment (richrace) [inline src/utils/editableFieldsValidationError.ts:24]: Is this meant to be here?

File: src/utils/editableFieldsValidationError.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+const editableFieldsValidationError = (courtCase: DisplayFullCourtCase, amendments: AmendmentRecords): boolean => {
+  const exceptions = courtCase.aho.Exceptions
+  if (exceptions.length === 0) {
+    return false
+  }
+
+  return (
+    // Object.entries(amendments).length === 0 ||
```
Comment (richrace) [inline src/utils/editableFieldsValidationError.ts:81]: Another comment

File: src/utils/editableFieldsValidationError.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+// return true when invalid value is entered
+
+///// If there is no exception
+// return false when field is empty
+// return true when invalid value is entered
+
+// if (amendments.asn) {
+//   return !isAsnFormatValid(amendments?.asn as string) || isEmptyOrWhiteSpace(amendments?.asn as string)
+// }
+const asnValidationError = (exceptions: Exception[], amendments: AmendmentRecords): boolean => {
```
Comment (richrace) [inline src/utils/editableFieldsValidationError.ts:25]: I would like if each error would in it's own file

**Review by richrace — CHANGES_REQUESTED** [review]

File: src/utils/editableFieldsValidationError.ts (hunk 1 above)
Comment (tausifpatel) [reply src/utils/editableFieldsValidationError.ts]: Removed comments

File: src/utils/editableFieldsValidationError.ts (hunk 3 above)
Comment (tausifpatel) [reply src/utils/editableFieldsValidationError.ts]: Separated into each file

**Review by richrace — APPROVED** [review]
LGTM 👍

**Pull request merged**